packages/hash/frontend/src/blocks/page/createEditorView.ts

```diff
@@ -123,10 +123,22 @@ export const createEditorView = (
 
   view.dom.classList.add(styles.ProseMirror);
 
-  for (const meta of preloadedBlocks) {
-    manager.defineNewBlock(meta);
+  // prosemirror will use the first node type (per group) for auto-creation.
+  // we want this to be the paragraph node type.
+  const blocksMetaArray = Object.values(blocksMeta);
+
+  const paragraphBlockMeta = blocksMetaArray.find(
+    (blockMeta) => blockMeta.componentMetadata.name === "paragraph",
```

Comment:
I think its name is now `@hashintel/block-paragraph` which is causing this issue

![image](https://user-images.githubusercontent.com/719814/150584574-d936ecdb-a86f-4da2-9947-526d9964f2bc.png)

Reply:
Same in tests: https://github.com/hashintel/hash/runs/4898525429?check_suite_focus=true

@espretto here's how you can inspect them: [internal Slack link](https://hashintel.slack.com/archives/C022217GAHF/p1642786720107700?thread_ts=1642771189.088000&cid=C022217GAHF).

Reply:
This assessment is correct and I've fixed it here: 5490e9af960fd8a61e1b7a2a61987d4b3adf1439
Any future name changes / sync problems between the above code and `blockPaths.json` will throw the above error.